libraries/core-react/src/Slider/Slider.tsx

```diff
@@ -144,8 +149,40 @@ const SrOnlyLabel = styled.label`
   white-space: nowrap;
   border-width: 0;
 `
+type SliderValueType = number[] | number
+
+export type Props = {
+  /** Id for the elements that labels this slider */
+  ariaLabelledby: string
+  /** Components value, range of numbers */
+  value: SliderValueType
+  /** Function to be called when value change */
+  onChange?: (
+    event: MouseEvent | KeyboardEvent,
+    newValue: SliderValueType,
+  ) => void
+  /* Function to be called when value is committed by mouseup event */
```

Comment:
It seems like only two ** works for enabling the `description` feature in props tables. Should probably be fixed in a lot of components...